composer/trainer/trainer.py

```diff
@@ -946,6 +974,17 @@ def __init__(
                     stream=console_stream,
                 ))
 
+        if checkpoint_save_path is not None:
+            object_store_logger = _create_object_store_logger_from_uri(uri=checkpoint_save_path)
+            if object_store_logger:
+                loggers.append(object_store_logger)
+
+        
+        load_object_store = _create_object_store_from_uri(uri=load_path) if load_path is not None else None
+        if isinstance(load_object_store, LoggerDestination):
```

Comment:
Why does this need to get added to the loggers?

Reply:
if it's a `WandBLogger`

Reply:
What I mean is, if the wandblogger is just an object store for loading a checkpoint, it doesn't actually need to be added to the loggers right? If I want a wandblogger for logging metrics and hyperparams I would specify that separately right?

Reply:
Ah I see. You're assuming that if a user passes a wandb:// path to load a checkpoint from, they also want wandb logging for their run.